### DISCUSSION TITLE
Focus button gives too much power to laser

### DISCUSSION THREAD

**Discussion by FrancescoF1945** [arkypita/LaserGRBL#1927] [General]
Hi, when I hit the focus button the spot has too much power and burns a spot onthe  surface.
Is there a way to change focusing power from stated 30% to something less?
Thanks in advance...

**Comment by LsrSal** [comment]
Default is not 30% but 3%, which is S-value 30.
My laser does not burn at 3% but certainly too bright to focus comfortably. I ave set it to 0.5% by editing first line:

Right click on the button and edit first line:
M3 S[$30*5/1000]
above will be dependent on your $30 max power settings, which for most systems is 1000. 
So usually you can just use:
M3 S5

**Comment by FrancescoF1945** [comment]
TNX for answering, LsrSal, was a typo, S30 = 3% of course...
I'll try 0.5%, good advice.
Just to chat, I searched in the code (not my language) but I've found nothing about the function called by the "Focus" button, otherwise it'd be easy change value and recompile...

**Reply by LsrSal** [reply]
I guess I was not clear: this is user configurable, no need to recompile. Right click over focus button and you will be able to edit commands for button actions.

**Comment by FrancescoF1945** [comment]
My bad, you're right... I feel dumb!
I'll read carefully next time.
Thank you again.
